checklists/alz_checklist.en.json

```diff
@@ -319,29 +298,6 @@
             "training": "https://learn.microsoft.com/learn/modules/azure-active-directory/",
             "link": "https://learn.microsoft.com/azure/architecture/reference-architectures/identity/adds-extend-domain#vm-recommendations"
         },
-        {
-            "category": "Identity and Access Management",
-            "subcategory": "Identity",
-            "text": "If Entra Domain Services in use, evaluate the compatibility of all workloads",
-            "waf": "Security",
-            "service": "Entra",
-            "guid": "8b9fe5c4-1049-4d40-9a92-3c3474d00018",
-            "id": "B03.12",
-            "severity": "Medium",
-            "training": "https://learn.microsoft.com/learn/modules/implement-hybrid-identity-windows-server/",
-            "link": "https://learn.microsoft.com/azure/active-directory-domain-services/overview"
-        },
-        {
-            "category": "Identity and Access Management",
```

Comment:
As part of removing this one, any problems with updating C02.13 to say the following:

If servers will be used for Identity services, like domain controllers, establish a dedicated identity subscription in the identity management group, to host these services.  Make sure that resources are set to use the domain controllers available in their region.

Reply:
I'll proceed with removal but will update C02.13 with comment around leveraging correct DC

Reply:
Sounds good!